Examples/Cocoapods/SpeechToText/Speech/SpeechRecognitionService.swift

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2019, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+import SwiftGRPC
+
+let API_KEY : String = "YOUR_API_KEY"
+let HOST = "speech.googleapis.com"
+
+final class Atomic<A> {
+  private let queue = DispatchQueue(label: "Atomic serial queue")
+  private var _value: A
+  init(_ value: A) {
+    self._value = value
+  }
+  
+  var value: A {
+    return queue.sync { self._value }
+  }
+  
+  func mutate(_ transform: (inout A) -> ()) {
```

Comment:
why not use a `set` method; it should be sufficient for our use. Also, consider giving credit to https://www.objc.io/blog/2018/12/18/atomic-variables/.